paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -208,6 +205,17 @@ def sync_all_secrets(
                 soa_dir=soa_dir,
             )
         )
+        results.append(
+            sync_crypto_secrets(
```

Comment:
I wonder if we should start pulling these out into separate jobs? If we keep adding things here it makes it harder to debug failures in my opinion. Also if this code has a bug and stalls/crashes then we might not be able to sync boto secrets or paasta secrets so it increases the reliability risk. If we had a separate cron job for each then we can run/monitor/alert on them separately.